src/containers/Home.jsx

```diff
@@ -1,14 +1,76 @@
-import React, { Component } from 'react';
+/* 
+Home page for SE Society
+Last Edited: Tony Zhao 2020-04-22
+*/
+
+import React, { Component, Fragment } from 'react';
+import { Button, Container, Jumbotron, Row, Col } from 'react-bootstrap';
+import EventList from '../components/EventList/EventList';
+import NewsList from '../components/NewsList';
+const Hero = () => (
+  <Jumbotron>
+    <Container>
+      <Row>
+        <Col sm>
+          {/* <PhotoReel />
+          TODO: Implement PhotoReel
+         */}
+        </Col>
+        <Col sm>
+          <h1>welcome</h1>
+          <p>
+            This is the official website of the UW Software Engineering Society.
+            A place where we [find a collection of english words that emulate
+            our desire to create events to support our program socially and
+            professionally]
+          </p>
+          <a href="/events">something about our latest move --></a>
```

Comment:
looks good! Can you take a sec and change this to be purple though, the blue doesn't really match

Reply:
Fixed!